easybuild/easyblocks/t/tbb.py

```diff
@@ -114,9 +123,15 @@ def build_step(self):
         """Configure TBB build/installation."""
         if self.toolchain.is_system_toolchain():
             IntelBase.build_step(self)
-        else:
-            # build with: make compiler={icl, icc, gcc, clang}
-            self.cfg.update('buildopts', 'compiler="%s"' % os.getenv('CC'))
+            return
```

Comment:
Not a big fan of inline `return`s, makes it more difficult to see that there's two different code paths in here, imho.

And this method is small enough that a simple `if/else` is clear?

Reply:
I'd say that's a question of style - I myself am not a fan of multiple nesting (2 or more), which this avoids. Rewritten anyway